packages/flutter/lib/src/material/page.dart

```diff
@@ -83,6 +84,14 @@ class MaterialPageRoute<T> extends PageRoute<T> {
     assert(opaque);
   }
 
+  /// Turns on the fading of routes during page transitions.
+  ///
+  /// This is currently disabled by default because of performance issues on
+  /// low-end phones. Eventually these issues will be resolved and this flag
+  /// will be removed.
+  @Deprecated('This flag will eventually be removed once the performance issues are resolved.')
```

Comment:
Include a link to the issue so people can follow it if they want to know when this gets removed?

Reply:
Added.